### PR TITLE
Feature/aave claim rewards

### PR DESCRIPTION
**Pull request by kjaredic**

### CODE REVIEW

File: test/aave/aave-claim-stkAave.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        dataProvider = await getAaveDataProvider();
+        aTokenInfo = await getAaveTokenInfo(dataProvider, assetInfo.address);
+    });
+
+    it(`... should supply ${supplyAmount} ${tokenSymbol} to Aave`, async () => {
+        // eslint-disable-next-line max-len
+        await supplyAave(proxy, AAVE_MARKET, hre.ethers.utils.parseUnits(supplyAmount, 18), WETH_ADDRESS, senderAcc.address);
+    });
+
+    it('... should claim accrued rewards', async () => {
```
Comment (nklipa13) [inline test/aave/aave-claim-stkAave.js:55]: Test sending 0 amount and test sending amount > unclaimedRewards

**Review by nklipa13 — CHANGES_REQUESTED** [review]

File: test/aave/aave-claim-stkAave.js (hunk 1 above)
Comment (kjaredic) [reply test/aave/aave-claim-stkAave.js]: done

Comment (kjaredic) [reply test/aave/aave-claim-stkAave.js]: done

**Review by nklipa13 — APPROVED** [review]

**Comment by nklipa13** [comment]
lgtm

**Pull request merged**